compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +157,73 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init();
+
+  // connect Custom(BroadcastTo) to graph
+  _g.addV2()->inputs(1, _g.custom_broadcastTo_out());
```

Comment:
purpose of `init()` is to do connection of test graphlet and `TestIOGraphlet`s.
how about adding two `init()` methods for connecting `_custom_broadcastTo` or `_builtin_broadcastTo` ?

Reply:
@seanshpark 
> how about adding two init() methods for connecting _custom_broadcastTo or _builtin_broadcastTo ?

I think it's a good idea. :)
Then, Would it be better to make a function that is set to `Builtin` or `Custom` according to a argument?

Reply:
> Then, Would it be better to make a function that is set to Builtin or Custom according to a argument?

That is also OK.

Reply:
@seanshpark 

I reflected comments in this PR :)